smartMeter/jsonld-contexts/smartMeter.jsonld

```diff
@@ -0,0 +1,45 @@
+{
+    "@context": {
+        "Consumer": "https://uri.fiware.org/ns/data-models#Consumer",
+        "minFlow": "https://uri.fiware.org/ns/data-models#minFlow",
+        "maxFlow": "https://uri.fiware.org/ns/data-models#maxFlow",
+        "persistenceFlowDuration": "https://uri.fiware.org/ns/data-models#persistenceFlowDuration",
+        "alarm": "https://uri.fiware.org/ns/data-models#alarm",
+        "acquisitionStageFailure": {
+            "@id": "https://uri.fiware.org/ns/data-models#acquisitionStageFailure",
+            "@type": "http://www.w3.org/2001/XMLSchema#boolean"
+        },
+        "alarmFlowPersistence": {
+            "@id": "https://uri.fiware.org/ns/data-models#alarmFlowPersistence",
+            "@type": "http://www.w3.org/2001/XMLSchema#boolean"
+        },
+        "alarmInProgressAlarms": {
```

Comment:
two times the `alarm` word is maybe too much?

Reply:
yes